scripts/release.sh

```diff
@@ -421,6 +421,18 @@ for crate in "${crates_to_publish[@]}"; do
     cleanup_actions+=('git reset -- '"$crate/Cargo.toml")
 done
 
+# Generate the nickel grammar so that users of the published crate don't
+# have to deal with lalrpop being slow.
+# 
+# cargo check is the cheapest supported way to run just the build script
+# https://github.com/rust-lang/cargo/issues/7178
+# The output path is hard to predict (it contains a hash), so we find it
+# using a clean output directory and some globbing
+rm -rf __temp_target
+cargo check -p nickel-lang-core --target-dir=__temp_target
+cp __temp_target/debug/build/nickel-lang-core-*/out/parser/grammar.rs core/src/parser/grammar.rs
+git add core/src/parser/grammar.rs
```

Comment:
should we remove the temporary directory? I guess its size might not be entirely negligible (and it will also pollute the git status).

Reply:
How do you feel about this now that we're using `mktemp`? I usually prefer to let my `/tmp/` get polluted because I have it configured to clean on boot (and not auto-removing things is sometimes useful for debugging).

Reply:
It might be worse with `mktemp` because if we run 10 time the script now we have 10 times the space usage (since it'd be a different, random dir each time). Although it should get erased at reboot at least. I would still vouch for deleting the directory once we're done; we won't use it anymore, it's a one line in the release script and should prevent most problems. Maybe we need to do something like `rm -rf $temp_dir || true` to avoid making the whole script fail if for any reason we can't delete it, as it's not a big deal.

Reply:
Sure, makes sense. I added it after the `cargo check` instead of putting it in `cleanup_actions`, so that if something fails exactly at that step then the output will be preserved.